### DISCUSSION TITLE
Managing arm64/ppc64le/s390x pipelines in CI and SSH servers

### DISCUSSION THREAD

**Discussion by junaruga** [jruby/jruby#8008] [Ideas]
Seeing the minor CPU architecture issues such as https://github.com/jruby/jruby/issues/8005, I suppose someone (not me) can maintain the pipelines by CI and SSH accessing servers sponsored by Arm and IBM. It's effective to test the pipelines in this project directly.

Here is the case. In the Ruby project (ruby/ruby), we are maintaining the arm64/ppc64le/s390x pipelines on the pull request by Travis CI, and SSH accessing Ubuntu arm64/ppc64le/s390x servers sponsored by Arm and IBM below. I am the person in charge of it.

* [Arm (Works on Arm)](https://github.com/WorksOnArm/equinix-metal-arm64-cluster), and the [landing page](https://www.arm.com/markets/computing-infrastructure/works-on-arm) for arm64
* [Oregon State University Open Source Hub Power Development Hosting](https://osuosl.org/services/powerdev/) for ppc64le
* [Oregon State University Open Source Hub IBM-Z Development Hosting](https://osuosl.org/services/ibm-z/) for s390x, or [LinuxOne Community Cloud](https://developer.ibm.com/articles/get-started-with-ibm-linuxone/) program. I am not sure about it.

You can check [this page](https://github.com/ruby/ruby/wiki/CI-Servers#travis-ci) for details.

**Comment by enebo** [comment]
@junaruga I think this is a great idea.
